app/views/categories/index.html.erb

```diff
@@ -1,14 +1,30 @@
 <p style="color: green"><%= notice %></p>
+<%= button_to "Log Out", destroy_user_session_path, class: "button is-link is-rounded is-normal is-danger", method: :delete, data:{turbo:false} %>
 
-<h1>Categories</h1>
-
-<div id="categories">
-  <% @categories.each do |category| %>
-    <%= render category %>
-    <p>
-      <%= link_to "Show this category", category %>
-    </p>
-  <% end %>
-</div>
-
-<%= link_to "New category", new_category_path %>
+  <h1 class="title is-3 has-text-centered mt-5">Categories</h1>
+  <div class="container">
+    <% @categories.each do |category| %>
+      <div class="block m-6 has-background-info-light p-5 ">
+        <div class="media-left">
+          <figure class="image is-64x64 margin-4">
+            <% if category.present? && category.image.attached? %>
+              <%= image_tag(category.image) %>
+            <% end %>
+          </figure>
+        </div>
+        <div class="media-content">
+          <strong>    <%= link_to category.name, category_entities_path(category) %>
+          </strong>
+        </div>
+        <p class="m-4">
+          <%= link_to "Show this category", category, class: "button is-danger is-rounded is-small
```

Comment:
- [ ] Upon clicking the "Show category" button I should be taken to the entities page for that category rather I am being taken to the show page of the category. Kindly fix that.